### PR TITLE
Change or remove punctuation mark from sentence

### PR DESCRIPTION
**Pull request by krivaten**
In using this library, I thought it would be nice to add in an option to change and/or remove the punctuation mark at the end of a sentence. This would make it ideal for generating random things like alert messages that you may want to have end with an exclamation mark instead of a period, or page titles where you may want no punctuation mark at all.

### CODE REVIEW

File: chance.js [hunk 1]
```diff
@@ -462,12 +462,19 @@
...
         text = word_array.join(' ');
-
-        // Capitalize first letter of sentence, add period at end
-        text = this.capitalize(text) + '.';
+        
+        // Capitalize first letter of sentence
+        text = this.capitalize(text);
+        
+        // Make sure punctuation has a usable value
+        if (punctuation !== false && !/^[\.\?;!:]$/.test(punctuation)) punctuation = '.';
```
Comment (somejeff) [inline chance.js]: As a guideline rule, even for one-liner conditions, you must still use brackets

File: chance.js [hunk 2]
```diff
@@ -462,12 +462,19 @@
...
+        
+        // Add punctuation mark
+        if (punctuation) text += punctuation;
```
Comment (somejeff) [inline chance.js]: As a guideline rule, even for one-liner conditions, you must still use brackets

**Comment by krivaten** [comment]
Updated the code per your comments. Thanks for the feedback!

**Comment by victorquinn** [comment]
Looking good and thanks for the cleanup!

**Pull request merged**